package.json

```diff
@@ -207,6 +207,10 @@
 		"onCommand:openshift.project.set",
 		"onCommand:openshift.project.delete",
 		"onCommand:openshift.project.delete.palette",
+		"onCommand:openshift.namespace.create",
+		"onCommand:openshift.namespace.set",
+		"onCommand:openshift.namespace.delete",
```

Comment:
Make sure to hide this entry from the command palette (this one expects a namespace name to be passed to the command, whereas the `.palette` one opens a quick pick)

Reply:
We actually didn't have any `openshift.*.delete.palette` - like command handlers registered.

So, I've hidden the mentioned `.delete` commands from Command Palette and added the according `.delete.palette` commands:

[Screencast from 2024-03-04 23-39-39.webm](https://github.com/redhat-developer/vscode-openshift-tools/assets/620781/547d3220-3917-4b49-92e6-386d3ccdb35c)